src/section-introduction-to-functions.ptx

```diff
@@ -248,7 +248,7 @@
                 <mrow>16+y^2\amp=25</mrow>
                 <mrow>y^2\amp=9</mrow>
             </md>
-            At this point, we see that <m>y</m> could be <m>3</m> <em>or</em> <m>y</m> could be <m>-3</m>. There are two <m>y</m>-values for the single <m>x</m>-value, so that must mean that <m>x^2+y^2=25</m> cannot represent a function.</p>
+            At this point, we see that <m>y</m> could be <m>3</m> <em>or</em> <m>y</m> could be <m>-3</m>. There are two <m>y</m>-values for the single <m>x</m>-value, so that must mean that <m>x^2+y^2=25</m> cMarianaot represent a function.</p>
```

Comment:
Look like a typo, but I'm not sure what the intent was. And one more instance below. Could you take a look, make a commit, and push that commit?

Reply:
Good catch! I forgot to check the red and green on github. That happened when I replaced "Ann" with "Mariana"